src/components/research/pills.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import { css } from "@emotion/core";
+import PropTypes from "prop-types";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faChevronUp, faChevronDown } from "@fortawesome/free-solid-svg-icons";
+import { mq, BreakPoint } from "../../util/mq";
+
+const withPrefixes = (style) => `
+    ${style}
+    -webkit-${style}
+    -moz-${style}
+    -ms-${style}
+`
+
+const cssPills = css`
+    ${withPrefixes`box-shadow: 0 0 4px 0 rgba(0,0,0,0.2);`}
+    ${mq(BreakPoint.md)} {
+        ${withPrefixes`box-shadow: none;`}
+    }
+`
+
+const cssListItem = css`
+    line-height: 2.5em;
+    border-top: solid 1px var(--light-grey);
+    display: block;
+
+    ${mq(BreakPoint.md)} {
+        border: none;
+        display: inline-block;
+    }
+`
+
+const Pill = ({ name, href }, key, cssPill) => (
```

Comment:
I'd be tempted to make this signature take everything in as properties on the first parameter so we can use `<Pill … />` later